prot.c

```diff
@@ -932,7 +938,8 @@ fmt_stats(char *buf, size_t size, void *x)
             srv->wal.nrec,
             srv->wal.filesize,
             id,
-            node_info.nodename);
+            node_info.nodename,
+            srv->draining);
```

Comment:
How about `drain_mode ? "true" : "false"` here instead?

And I suggest putting this before the id field instead of after, so there's less noise in the diff.